### ISSUE TITLE
Error when trying text-based model with single parameter

### ISSUE THREAD

**Issue by headmeister** [IMTtugraz/PyQMRI#6]
I encountered this possible bug when trying to fit the simple MonoExp model from the example.
When doing so, the python throws an error in the file GeneralModel.py on line 100 (for mypar in modelpar:). The error states that the variable modelpar appears to be non-iterable. After I tried adding an additional "fake" parameter "K" to the modelfile.

The model file looked then like this:

[MonoExp]

parameter = TE K

unknowns = M0 A1

signal = M0*exp(-TE*A1)*K

box_constraints_lower = 0,0

box_constraints_upper = 100,1

real_value_constraints = False,True

guess = 1,0.01

rescale = M0,1/A1

estimate_individual_phase = False

Then I set the "K" parameter to 1 in the .h5 file. 

This solved the issue, no more non-iterable error showed. It seems like parsing only one parameter creates some non-iterable structure and then it fails in setting the bounds for the for-loop. This will probably hurt a few people, since most models will have more parameters (TR, TE,TI...) but still, it might cause someone a headache.

Best regards

Jiri Vitous

**Comment by MaierOli2010** [comment]
Thank you for pointing to this bug.

The error is related to the structure that is read from the file.
If just one parameter is parsed, it will be returned as non-iterable, single symbolic value in line 64, otherwise as tuple of symbolics. 

A quick workaround is to add:

```
if type(modelpar) is not tuple:
            modelpar = [modelpar]
```

prior to line 100 but I need to check if this captures all cases. 

Until the next release fixes this, your workaround should also do the trick for others.

Cheers,
Oliver

**Comment by MaierOli2010** [comment]
Should be fixed by the latest merge to master [1a9418a](https://github.com/IMTtugraz/PyQMRI/commit/1a9418a9f5850e411827abe13b2845c4c93ba93d)
